packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -1,9 +1,9 @@
 import { AmountMath, AssetKind } from '@agoric/ertp';
 import { makeRatio } from '@agoric/zoe/src/contractSupport/index.js';
 import { deeplyFulfilledObject } from '@agoric/internal';
-import { Stable } from '@agoric/vats/src/tokens.js';
 import { E } from '@endo/far';
 import { parseRatio } from '@agoric/zoe/src/contractSupport/ratio.js';
+import { Stable } from '../tokens.js';
```

Comment:
Interesting... I thought this (moving tokens to this package) would make zoe depend on inter-protocol, but now I remember... we pass the token name in as part of `feeIssuerConfig`... I think that's a result of the RUN->IST rename.

Consider moving the `feeIssuerConfig` from `packages/vats/src/core/utils.js` to `inter-protocol` as well.

Reply:
I looking into moving `feeIssuerConfig` but it's consumed only in `@agoric/vats` so moving wouldn't help the dependency graph.